internal/compiler/object_tree.rs

```diff
@@ -741,10 +741,21 @@ impl Element {
                 ..
             } = r.lookup_property(&unresolved_prop_name);
             if !matches!(maybe_existing_prop_type, Type::Invalid) {
-                diag.push_error(
-                    format!("Cannot override property '{}'", prop_name),
-                    &prop_decl.DeclaredIdentifier().child_token(SyntaxKind::Identifier).unwrap(),
-                )
+                if matches!(maybe_existing_prop_type, Type::Callback { .. }) {
```

Comment:
You could siplify this by putting this before the parent if and do a else if.  or merge it with the parent if, in a match.

Reply:
Good idea.